great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -1727,8 +1732,8 @@ def _chart_domain_values(
         subtitle: Optional[str],
     ) -> Optional[alt.Chart]:
         implemented_metrics: Set[str] = {
-            "table_row_count",
-            "column_distinct_values_count",
+            sanitize_parameter_name(metric)
+            for metric in self.EXPECTATION_METRIC_MAP.values()
         }
```

Comment:
No longer hard-coding values - we can iterate on this moving forward.

Reply:
Nice!